src/Microsoft.SqlTools.ServiceLayer/SchemaCompare/SchemaCompareService.cs

```diff
@@ -227,6 +229,41 @@ await requestContext.SendResult(new SchemaCompareOptionsResult()
             }
         }
 
+        /// <summary>
+        /// Handles schema compare open SCMP request
+        /// </summary>
+        /// <returns></returns>
+        public async Task HandleSchemaCompareOpenScmpRequest(SchemaCompareOpenScmpParams parameters, RequestContext<SchemaCompareOpenScmpResult> requestContext)
+        {
+            try
+            {
+                Task schemaCompareTask = Task.Run(async () =>
```

Comment:
Can you please use the currentTask object in class - that way when we test service call.. we can wait on that to ensure this completes before test exits